### PR TITLE
Goa 2929 tsv header default

### PR DESCRIPTION
**Pull request by TonyWardell**
Only the default columns (as shown on the web application) will be included on the TSV output, if no selectedFields parameter is not used.

### CODE REVIEW

**Comment by TonyWardell** [comment]
Changes made.
I've removed the gene product type from the default field/header list, as it is not in the list of default fields that appear in the front end,
Taxon name and gene product name and gene product synonyms I'd removed previously in this branch.

File: annotation-rest/src/test/java/uk/ac/ebi/quickgo/annotation/download/converter/AnnotationToTSVTest.java [hunk 1]
```diff
@@ -318,17 +310,40 @@ public void onlyGetRequestedColumn() {
...
+//        private static final int COL_NAME = ;
+        private static final int COL_ECO_ID = 6;
+        private static final int COL_GO_EVIDENCE = 7;
+        private static final int COL_REFERENCE = 8;
+        private static final int COL_WITH = 9;
+        private static final int COL_TAXON = 10;
+        private static final int COL_ASSIGNED_BY = 11;
+        private static final int COL_ANNOTATION_EXTENSION = 12;
+        private static final int COL_DATE = 13;
+//        private static final int COL_TAXON_NAME = ;
```
Comment (eddturner) [inline annotation-rest/src/test/java/uk/ac/ebi/quickgo/annotation/download/converter/AnnotationToTSVTest.java]: rm commented out code

File: annotation-rest/src/test/java/uk/ac/ebi/quickgo/annotation/download/converter/AnnotationToTSVTest.java [hunk 2]
```diff
@@ -318,17 +310,40 @@ public void onlyGetRequestedColumn() {
...
+
+    private class DefaultColumnsWithSlimming{
+        private static final int COL_GENE_PRODUCT_DB = 0;
+        private static final int COL_GENE_PRODUCT_ID = 1;
+        private static final int COL_DB_OBJECT_SYMBOL = 2;
+        private static final int COL_QUALIFIER = 3;
+        private static final int COL_GO_ID = 4;
+        private static final int COL_SLIMMED_FROM = 5;
+        private static final int COL_GO_ASPECT = 6;
+//        private static final int COL_NAME = ;
```
Comment (eddturner) [inline annotation-rest/src/test/java/uk/ac/ebi/quickgo/annotation/download/converter/AnnotationToTSVTest.java]: rm commented out code

File: annotation-rest/src/test/java/uk/ac/ebi/quickgo/annotation/download/converter/AnnotationToTSVTest.java [hunk 3]
```diff
@@ -318,17 +310,40 @@ public void onlyGetRequestedColumn() {
...
-    private class NonSlimmedColumns{
+    private class DefaultColumns{
         private static final int COL_GENE_PRODUCT_DB = 0;
         private static final int COL_GENE_PRODUCT_ID = 1;
         private static final int COL_DB_OBJECT_SYMBOL = 2;
         private static final int COL_QUALIFIER = 3;
         private static final int COL_GO_ID = 4;
         private static final int COL_GO_ASPECT = 5;
-        private static final int COL_NAME = 6;
+//        private static final int COL_NAME = ;
```
[the remark this answers is not in the document]
Comment (TonyWardell) [reply annotation-rest/src/test/java/uk/ac/ebi/quickgo/annotation/download/converter/AnnotationToTSVTest.java]: Done.

**Pull request merged**